api/app/utils/cffdrs.py

```diff
@@ -0,0 +1,109 @@
+""" This module contains functions for computing fire weather metrics.
+"""
+import logging
+from rpy2.robjects.packages import importr
+cffdrs = importr('cffdrs')
```

Comment:
This may be a good candidate for a singleton pattern, if only to ensure that it's only loaded when actually needed.